docs/spark-getting-started.md

```diff
@@ -69,7 +69,7 @@ To create your first Iceberg table in Spark, use the `spark-sql` shell or `spark
 
 ```sql
 -- local is the path-based catalog defined above
-CREATE TABLE local.db.table (id bigint, data string) USING iceberg
+CREATE TABLE local.db.table (id bigint, data string) USING iceberg;
```

Comment:
Can you please look up ```sql in the code and fix all the places. I found few more places in the docs like this.

Reply:
Sure, will do.

Reply:
@ajantha-bhat, this is done.

Reply:
Would be even better to have some kind of linter for this

Reply:
@Fokko Agreed.